unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1933,15 +1855,14 @@ doDisplay outputLoc names tm = do
       loadTypeOfTerm' (Referent.Ref (Reference.DerivedId r))
         | Just (_, _, ty) <- Map.lookup r tms = pure $ Just (void ty)
       loadTypeOfTerm' r = fmap (fmap void) . Cli.runTransaction . Codebase.getTypeOfReferent codebase $ r
-  rendered <-
-    DisplayValues.displayTerm
-      ppe
-      loadTerm
-      loadTypeOfTerm'
-      evalTerm
-      loadDecl
-      tm
-  Cli.respond $ DisplayRendered loc rendered
+  rendered <- DisplayValues.displayTerm ppe loadTerm loadTypeOfTerm' evalTerm loadDecl tm
+  mayFP <- case outputLoc of
+    ConsoleLocation -> pure Nothing
+    FileLocation path -> Just <$> Directory.canonicalizePath path
+    LatestFileLocation -> traverse Directory.canonicalizePath $ fmap fst (loopState ^. #latestFile) <|> Just "scratch.u"
+  whenJust mayFP \fp -> do
+    liftIO $ writeSource (Text.pack fp) (Cli.PrependSource False $ Text.pack . P.toPlain 80 $ rendered)
+  Cli.respond $ DisplayRendered mayFP rendered
```

Comment:
This is to untangle later 😄

Reply:
Not sure what you mean?

Reply:
I don't remember now either; sorry about that. May have been something to do with eliminating `outputLoc`, but not for this PR in any case.